modules/client/src/main/scala/ch/epfl/bluebrain/nexus/admin/client/config/AdminConfig.scala

```diff
@@ -0,0 +1,5 @@
+package ch.epfl.bluebrain.nexus.admin.client.config
+
+import akka.http.scaladsl.model.Uri
+
+final case class AdminConfig(baseUri: Uri)
```

Comment:
This shouldn't matter if we attempt to append the path with a bit of care:
This is already present in `ProjectAclRoutes`, it just needs to be extracted and put alone in a class in `core` or whatever and then it can be used form anywhere else

```scala
 implicit class UriSyntax(uri: Uri) {
    def append(path: Path): Uri =
      uri.copy(path = (uri.path: Path) ++ path)
  }
```